src/test/libradosstriper/rados-striper.sh

```diff
@@ -77,10 +78,14 @@ function run() {
     rados --pool rbd listxattr toyfile.0000000000000000 | grep -v striper > $dir/xattrlist2 || return 1
     diff -w $dir/xattrlist2 $dir/reflist || return 1    
     rados --pool rbd --striper rmxattr toyfile somexattr || return 1
+
+    local attr_not_found_str="No data available"
+    [ `uname` = FreeBSD ] && \
+        attr_not_found_str="Attribute not found"
     rados --pool rbd --striper getxattr toyfile somexattr >& $dir/rmxattrerror
-    grep -q 'No data available' $dir/rmxattrerror || return 1
+    expect_fail grep -q "$attr_not_found_str" $dir/rmxattrerror 
```

Comment:
@tchaikov 
'MMM I guess we've upgrade expect_failure to do much more than in outer places.

Maybe we should run the script under the -e flag.
That would remove a lot of the `return 1` uglyfication